pkg/infra/teams/team0/init.go

```diff
@@ -23,9 +23,13 @@ var InitAgentMap = map[commons.ID]func() agent.Strategy{
 	// "CowardlyAgent": NewProbabilisticAgent(0.9, 0.05, 0.05),
 }
 
-func InitAgents(defaultStrategyMap map[commons.ID]func() agent.Strategy, gameConfig config.GameConfig, ptr *state.View) (numAgents uint, agentMap map[commons.ID]agent.Agent, agentStateMap map[commons.ID]state.AgentState) {
+func InitAgents(defaultStrategyMap map[commons.ID]func() agent.Strategy, gameConfig config.GameConfig, ptr *state.View) (numAgents uint, agentMap map[commons.ID]agent.Agent, agentStateMap map[commons.ID]state.AgentState, inventoryMap state.InventoryMap) {
 	agentMap = make(map[commons.ID]agent.Agent)
 	agentStateMap = make(map[commons.ID]state.AgentState)
+	inventoryMap = state.InventoryMap{
+		Weapons: make(map[commons.ItemID]uint),
+		Shields: make(map[commons.ItemID]uint),
+	}
```

Comment:
This might be for a future PR - but this will probably end up changing for agent's views of the loot since my guess is that most agents will sort both data structures prior to making any decisions or allocations.

Reply:
This inventoryMap contains all obtained items from loot, I thought it need to be hidden to agents?